src/rna.jl

```diff
@@ -43,7 +43,8 @@ function rna(args...; kwargs...)
     return Dict{Symbol, Float64}(
         :density => density(graph),
         :transitivity => global_clustering_coefficient(graph),
-        :averagepath => mean(map(x-> isinf(x) ? 0. : x, 1 ./ closeness_centrality(graph))),
+        :averagepath => sum(map(x-> x == typemax(x) ? 0. : x,
+                        floyd_warshall_shortest_paths(graph).dists)) / (nv(graph) * (nv(graph) - 1)),
```

Comment:
I do not think this is clear source code. So many operations in one line of code, some of which are not transparent to me. Can you please clarify the source a bit, and separate this into more steps?

Reply:
Sure, the source for this is in [Donner, 2010](https://iopscience.iop.org/article/10.1088/1367-2630/12/3/033025) pg. 18 eq. 26 (ref 1 in the rna.jl file).

I agree, that's a dense function. All it does is get the path lengths from LightGraphs, maps the Inf values to zero (as in Donner 2010) and then divided the sum by (N* (N-1)). If I break it down into multiple lines where would you like function to go?

Reply:
Just define a function after `rna` and call this function in the `:averagepath` mapping. Especially since you have a specific equation `Donner, 2010 pg. 18 eq. 26`, this should be referenced as a comment in the function :)

Reply:
OK. I've implemented that. New benchmarking is only slightly slower.

    BenchmarkTools.Trial: 
      memory estimate:  2.06 KiB
      allocs estimate:  8
      --------------
      minimum time:     1.060 μs (0.00% GC)
      median time:      1.125 μs (0.00% GC)
      mean time:        1.348 μs (5.33% GC)
      maximum time:     207.975 μs (98.31% GC)
      --------------
      samples:          10000
      evals/sample:     10